### PR TITLE
Untangle: Rename the Upgrades to the WordPresscom menu

### PR DESCRIPTION
**Pull request by arthur791004**
Related to https://github.com/Automattic/dotcom-forge/issues/5308

## Proposed changes:

* This PR moves items in the Upgrades menu to the new WordPress.com menu

| Before | After |
| - | - |
| image | image |

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Head to Settings → Hosting Configuration.
* Scroll to the Admin interface style section, and select Classic style.
* Refresh the page if needed.
* Ensure that the Upgrades menu is no longer present in the sidebar. Instead, make sure you can see the new WordPress.com menu, and ALL the items under the Upgrades menu are moved to the new menu.

### CODE REVIEW

**Comment by fushar** [comment]
I think this conflicts with https://github.com/Automattic/jetpack/pull/35365... cc: @candy02058912

**Comment by arthur791004** [comment]
I created a feature branch called `untangle/add-wpcom-menu`, and we can merge to PR related to the new WordPress.com menu there. With this feature branch, we can test the new menu via Jetpack Beta Tester. If everything works well, then we just merge the feature branch to the trunk 🙂

File: projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php [hunk 1]
```diff
@@ -126,6 +132,35 @@ public function add_stats_menu() {
 		add_menu_page( __( 'Stats', 'jetpack' ), __( 'Stats', 'jetpack' ), 'view_stats', 'https://wordpress.com/stats/day/' . $this->domain, null, 'dashicons-chart-bar', 3 );
 	}
 
+	/**
+	 * Adds WordPress.com menu.
+	 */
+	public function add_wpcom_menu() {
+		if ( get_option( 'wpcom_is_staging_site' ) ) {
```
Comment (fushar) [inline projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php:139]: Why is this needed?

Comment (candy02058912) [reply projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php:139]: I think to maintain the current behavior for the staging site since there isn't an Upgrade menu
staging site

I think bringing back the WordPress.com menu can be addressed in a separate PR.

**Review by candy02058912 — APPROVED** [review]
Tested and LGTM, we can merge this one first to get the momentum going.

File: projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php (hunk 1 above)
Comment (arthur791004) [reply projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php:139]: Yep. I didn't show the Upgrades menu for the staging site, so I'd prefer to keep it hidden for now. When we have more and more items in the WordPress.com menu, we can remove this check.

Comment (fushar) [reply projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php:139]: TIL about this, thanks for explaining!
